docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -269,13 +318,13 @@ Here's an example of connecting to Temporal Cloud from the CLI using an environm
 export TEMPORAL_API_KEY=<key secret>
 temporal workflow list --address <endpoint> \
 --namespace <namespace.accountid> \
---tls --tls-disable-host-verification \
 --grpc-meta "temporal-namespace=<namespace.accountid>"
+--tls
 ```
 
 ### SDK
 
-To use your API Key with a Temporal SDK, use headers to specify a Bearer token is provided for authentication and the Namespace for request routing.
+To use your API Key with a Temporal SDK, use headers to specify a Bearer token and a Namespace. The Bearer token header is used to indicate the authentication type of the request and the Namespace header is used for request routing.
```

Comment:
Is there a more specific way to refer to "headers"?   Something like "When using API keys, add your Bearer token to the Authorization header to authenticate your Temporal Cloud requests." And then something about setting the Namespace header for request routing?

Reply:
Also, earlier in the doc (not in the changes), you refer to KMS. Can you specify that's a key management system?